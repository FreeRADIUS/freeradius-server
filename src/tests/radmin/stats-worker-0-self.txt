stats worker 0 self
