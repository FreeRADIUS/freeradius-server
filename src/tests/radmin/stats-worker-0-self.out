count.in			0
count.out			0
count.dup			0
count.dropped			0
count.naks			0
count.active			0
count.runnable			0
cpu.request_time_rtt		0.000000000
cpu.average_request_time	0.000000000
cpu.used			0.000000
cpu.waiting			0.000
